### PR TITLE
games-strategy/widelands: version bump

### PR DESCRIPTION
**Pull request by P424D0X**

### CODE REVIEW

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit eutils versionator cmake-utils
+
+MY_PV=build$(get_version_component_range 2)
+MY_P=${PN}-${MY_PV}-src
+DESCRIPTION="A game similar to Settlers 2"
+HOMEPAGE="http://www.widelands.org/"
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: Please switch to https

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+SRC_URI="https://launchpad.net/widelands/${MY_PV}/${MY_PV}/+download/${MY_P}.tar.bz2"
+
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+DEPEND="dev-lang/lua:0
+	>=dev-libs/boost-1.48
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: Please append :=

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	media-libs/glew
+	media-libs/libpng:0
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: Please append :=

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	local mycmakeargs=(
+		'-DWL_VERSION_STANDARD=true'
+		"-DWL_INSTALL_BASEDIR=${PREFIX}"
+		"-DWL_INSTALL_DATADIR=${PREFIX}"
+	)
+	cmake-utils_src_configure
+}
+
+src_compile() {
+	cmake-utils_src_compile
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: you can remove src_compile entirely

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+src_install() {
+	cmake-utils_src_install
+	newicon data/images/logos/wl-ico-128.png ${PN}.png
+	make_desktop_entry ${PN} Widelands
+	dodoc ChangeLog CREDITS
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: Please use global DOCS array instead

File: games-strategy/widelands/widelands-0.19.ebuild (hunk 3 above)
Comment (SoapGentoo) [inline games-strategy/widelands/widelands-0.19.ebuild]: specifically, `:0=`

File: games-strategy/widelands/widelands-0.19.ebuild (hunk 5 above)
Comment (SoapGentoo) [inline games-strategy/widelands/widelands-0.19.ebuild]: not needed, both `ChangeLog` and `CREDITS` are part of einstalldocs default set

File: games-strategy/widelands/widelands-0.19.ebuild (hunk 3 above)
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: indeed

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+	local mycmakeargs=(
+		'-DWL_VERSION_STANDARD=true'
+		"-DWL_INSTALL_BASEDIR=${PREFIX}"
+		"-DWL_INSTALL_DATADIR=${PREFIX}"
+	)
+	cmake-utils_src_configure
+}
+
+src_install() {
+#	DOCS="ChangeLog CREDITS" cmake-utils_src_install
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: That was messed up a bit, please keep cmake-utils_src_install

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+DEPEND="dev-lang/lua:0
+	>=dev-libs/boost-1.48:=
+	media-libs/glew
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: I forgot, this also needs :=

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+EAPI=6
+inherit eutils versionator cmake-utils
+
+MY_PV=build$(get_version_component_range 2)
+MY_P=${PN}-${MY_PV}-src
+DESCRIPTION="A game similar to Settlers 2"
+HOMEPAGE="https://www.widelands.org/"
+SRC_URI="https://launchpad.net/widelands/${MY_PV}/${MY_PV}/+download/${MY_P}.tar.bz2"
+
+
```
Comment (a17r) [inline games-strategy/widelands/widelands-0.19.ebuild]: let's make repoman happy, it complains about an unnecessary empty line here.

If you use `repoman full` before you commit, or even better, `repoman commit` then these things are checked already locally before you push.

File: games-strategy/widelands/widelands-0.19.ebuild [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+S=${WORKDIR}/${MY_P}
+
+CMAKE_BUILD_TYPE=Release
+
+# Game is NOT happy being moved from /usr/share/games
+PREFIX="/usr/share/games/${PN}"
+
+src_prepare() {
+	default
```
Comment (SoapGentoo) [inline games-strategy/widelands/widelands-0.19.ebuild]: use `cmake-utils_src_prepare` instead of `default` when using `cmake-utils`

**Comment by SoapGentoo** [comment]
@P424D0X please rebase **not** merge your PR. Have a look at https://wiki.gentoo.org/wiki/Gentoo_Github#Rebasing_commits how to squash and rebase your commits. All of this should be one commit. In the Gentoo contribution workflow, you will never need to merge stuff, but rebase often.

**Comment by SoapGentoo** [comment]
Reopen once you've done the squashing.

**Pull request closed without merging**